### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**
Initial PR

### CODE REVIEW

**Review by xansec — CHANGES_REQUESTED** [review]
There's no need to include FuzzedDataProvider.h here, it's part of the library in the latest LLVM. Either pull the latest LLVM or install from source. Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Fixed!

File: docs/reference/cwk_path_join.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: cwk_path_join
 description: Joins two paths together.
```
Comment (xansec) [inline docs/reference/cwk_path_join.md]: not sure why this is showing as modified?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: docs/reference/cwk_path_join.md (hunk 1 above)
Comment (capuanob) [reply docs/reference/cwk_path_join.md]: @xansec  I'm not sure... Resolved

**Pull request merged**